bazel/test.bzl

```diff
@@ -376,7 +379,27 @@ def redpanda_cc_bench(
         features = [
             "layering_check",
         ],
-        tags = resource_tags + tags,
+        tags = tags,
         env = env,
         data = data,
     )
+    write_file(
+        name = name + "_test_script",
+        out = name + "_test_wrapper.sh",
+        content = [
+            "#!/bin/bash",
+            "exec $@ --iterations=1 --runs=1 --duration=0 --no-stdout",
```

Comment:
Do we not need seastar args here or does that get pulled in somewhere different?